lib/CUDD/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
 set(SOURCES
        cuddAddNeg.c
        cuddBddCorr.c
-       cuddDecomp.c
+       cuddDecomp.cpp
```

Comment:
could you remove the cmake folder and re-run? that's the issue

i renamed the `c` files to `cpp` because they reference `std11` elements